### ISSUE TITLE
inner folders/files not loading

### ISSUE THREAD

**Issue by Yuvaldv** [astrada/google-drive-ocamlfuse#350]
main files/folders load onto mounted drive,
but i have to scrape each and every folder to load the rest of the files-- they don't load automatically
literally have to open the folder, wait a couple seconds/minutes for the files to load.
Is there a way for force sync?

**Comment by astrada** [comment]
No, sorry, this isn't a sync app. You should consider using https://github.com/Grive/grive or https://github.com/odeke-em/drive.
